### PR TITLE
Fix up matcher in http-refresh-token service

### PR DESCRIPTION
**Pull request by bhollis**
Small cleanup, including a fix to a regex that wasn't quite right.

### CODE REVIEW

File: src/scripts/oauth/http-refresh-token.service.js [hunk 1]
```diff
@@ -8,22 +8,15 @@ function HttpRefreshTokenService($rootScope, $q, $injector, OAuthService, OAuthT
 
   const service = this;
   let cache = null;
-  const limiters = [
-    /www\.bungie.net\/Platform\/User\/*/,
-    /www\.bungie.net\/Platform\/Destiny\/*/
-  ];
+  const matcher = /www\.bungie.net\/Platform\/(User|Destiny)\//;
```
Comment (kyleshay) [inline src/scripts/oauth/http-refresh-token.service.js]: Don't need the wild card at the end? Is that what was "not right"? I like this regex vs the array, much cleaner! 👍

Comment (kyleshay) [inline src/scripts/oauth/http-refresh-token.service.js]: need to escape the `.` before `.net`?

Comment (kyleshay) [inline src/scripts/oauth/http-refresh-token.service.js]: Sorry for the multi-line. Just to clarify: do we want to force the URL to have a trailing slash at the end, case sensitivity?

Comment (bhollis) [inline src/scripts/oauth/http-refresh-token.service.js]: You're right, the dot needs escaping. The star before matched zero or more slashes, which didn't seem useful. I think case sensitivity is all right, and yeah we want there to be a slash after User or Platform.

**Pull request merged**